incubator/binding-asyncapi.spec/src/main/scripts/io/aklivity/zilla/specs/binding/asyncapi/config/kafka/asyncapi.yaml

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
This one seems to be the standard v3 kafka example, lets we use this instead if possible.

https://github.com/asyncapi/spec/blob/master/examples/streetlights-kafka-asyncapi.yml